WalletWasabi/Helpers/PowerSaving/MacOsInhibitorTask.cs

```diff
@@ -0,0 +1,33 @@
+using System.Diagnostics;
+using WalletWasabi.Logging;
+using WalletWasabi.Microservices;
+
+namespace WalletWasabi.Helpers.PowerSaving;
+
+/// <summary>
+/// Inhibitor based on <c>caffeinate</c> command.
+/// </summary>
+public class MacOsInhibitorTask : BaseInhibitorTask
+{
+	/// <remarks>Use the constructor only in tests.</remarks>
+	internal MacOsInhibitorTask(TimeSpan period, string reason, ProcessAsync process)
+		: base(period, reason, process)
+	{
+	}
+
+	public static MacOsInhibitorTask Create(TimeSpan basePeriod, string reason)
+	{
+		// TODO: Make sure that the caffeinate is terminated once the parent process (WW) finishes.
```

Comment:
The process `caffeinate` is terminated once WW finishes.

Reply:
Yes. In the Activity Monitor the process only stays few tens of seconds even when WW is running.
When I force quit Wasabi, `caffeinate` stays some seconds and then finishes